test/tests/functional/pbs_dup_acc_log_for_resv.py

```diff
@@ -51,6 +52,52 @@ def setUp(self):
             self.skipTest('test requires two MoMs as input, ' +
                           'use -p moms=<mom1>:<mom2>')
 
+    def check_log_resv_time(self, accounting_log_msg, start_tup,
+                            end_tup, check_status):
+        """
+        This is a helper function to reterive start and end time
+        of standing rervation from accounting log.
+        """
+        find_start = m[start_tup][end_tup].find("start=")
+        find_end = m[start_tup][end_tup].find("end=")
+        find_duration = m[start_tup][end_tup].find("duration=")
+        if check_status == "start":
+            time_str = m[start_tup][end_tup][
+                find_start:find_start + (find_end - find_start - 1)]
+            new_time_str = time_str.replace("\'", "")
```

Comment:
I don't see any single quotes in the 'B' record.  Why are you stripping them out?